typescript/packages/subsurface-viewer/src/layers/grid3d/fragment.fs.glsl.ts

```diff
@@ -52,9 +58,8 @@ vec4 getContinuousPropertyColor (float propertyValue) {
 vec4 getDiscretePropertyColor (float propertyValue) {
 
    vec4 color = vec4(1.0, 1.0, 1.0, 1.0);
-   float tolerance = (1.0 / colorMapSize) * 0.5;
-     
-   if (propertyValue < colorMapRangeMin - tolerance || propertyValue > colorMapRangeMax + tolerance) {
+    
+   if (propertyValue < colorMapRangeMin || propertyValue > colorMapRangeMax) {
```

Comment:
Have you checked that discrete property grids with clamping are colored correctly and there is no "color fighting"?

Reply:
I added the tolerances back in for this layer.  Im not quite sure why they are needed in the continuous case.